daisy_workflows/image_import/import_image.sh

```diff
@@ -72,13 +77,24 @@ function copyImageToScratchDisk() {
   mkdir -p /daisy-scratch
   # /dev/sdb is used since the scratch disk is the second
   # disk that's attached in import_disk.wf.json.
-  mkfs.ext4 /dev/sdb
+  #
+  # We disable reserved blocks to save disk space via `-m 0`. Typically
+  # this is 5% and we won't be using it.
+  mkfs.ext4 /dev/sdb -m 0
   mount /dev/sdb /daisy-scratch
   if [[ $? -ne 0 ]]; then
     echo "ImportFailed: Failed to prepare scratch disk."
   fi
-  gsutil cp "${SOURCE_URL}" "${IMAGE_PATH}"
-  echo "Import: Copied image from ${SOURCE_URL} to ${IMAGE_PATH}"
+
+  # Standard error for `gsutil cp` contains a progress meter that when written
+  # to the console will exceed the logging daemon's buffer for large files.
+  # The end of the stream may contain useful debugging messages, hence the
+  # usage of `tail` when we echo the error inside this block.
+  if ! out=$(gsutil cp "${SOURCE_URL}" "${IMAGE_PATH}" 2> gsutil.cp.err); then
+    echo "ImportFailed: Failed to download image to scratch [Privacy-> from ${SOURCE_URL} to ${IMAGE_PATH}, error: $(tail gsutil.cp.err) <-Privacy]"
```

Comment:
There is a problem here, I think: "StatusMatch", "SuccessMatch" and "FailureMatch" only matches single line. "tail" may give multiple lines.

Reply:
Interesting. In my testing I thought it was working, but looking at it again, it's not. I'll iterate over the lines in the error log and explicitly print the ones that I want to keep.